### ISSUE TITLE
Python LSP repos which have been archived

### ISSUE THREAD

**Issue by nielsbom** [langserver/langserver.github.io#302]
The repo for [Microsoft/python-language-server](https://github.com/Microsoft/python-language-server) has been archived since November 2022. I guess that means it will no longer be developed or receive support. Shall we add that info?

The repo for [sourcegraph/python-langserver](https://github.com/sourcegraph/python-langserver) has also been archived since November 2022. I guess that means it will no longer be developed or receive support. Shall we add that info?

It's also possible to use the [Github API](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#get-a-repository) to check whether a repo has been archived and then automatically update the contents of this site with that information.

**Comment by jdorfman** [comment]
@keegancsmith whatcha think?

**Comment by keegancsmith** [comment]
Yeah we should indicate they are archived. Building something which detects this can probably done with a little bit of code, but my main concern is where it would run and how we would see it. So I'd just rely on humans noticing and updating this repo.

**Comment by jdorfman** [comment]
Thanks @keegancsmith 

@nielsbom something like this?

![Image 2023-01-13 at 2 25 54 PM](https://user-images.githubusercontent.com/398230/212430629-a8ab1df7-2e61-41e6-9bab-57027fcb4fc3.jpg)

**Comment by nielsbom** [comment]
Looks good @jdorfman .

If the site is a Github Pages website you could use Github Actions to check archived-ness of all the repos.

**Comment by goyalyashpal** [comment]
hi!

there are 2 issues getting mingled here:
1. how to show the archive status to the known archived projects (the screenshot by @ jdorfman above)
2. how to know which projects are archived (the github actions thingy)

I would say to first tackle the (1.) issue - figuring out how to show the status for known repos.

the automation part (2.) is not that urgent - it is not even required, as it can even be done on a crowdsourcing basis; but unless (1.) is finalised, (2.) won't do _any_ help

**Comment by jdorfman** [comment]
@goyalyashpal I think for now I will create a PR with the proposed solution (screenshot) and close out this issue. Then you or someone can create a new issue. I'm leaning toward manual/crowdsourcing since it is a rare occurrence.

**Comment by jdorfman** [comment]
Closing this out, @goyalyashpal thanks for the push.

**Comment by goyalyashpal** [comment]
awesome, glad i could be of help (even just for push) :) 
🤝
